<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRecipeForm</class>
 <widget class="QWidget" name="AddRecipeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddRecipe</string>
  </property>
  <widget class="QLabel" name="nameTitle">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ingidientLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="descriptionTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>361</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
